src/Frontend/Page/Idea.hs

```diff
@@ -384,17 +384,17 @@ createOrEditPage showDeleteButton cancelUrl v form p = semanticDiv p $ do
                             i_ [class_ "icon-trash-o"] nil
                             "Idee löschen"
 
-
 instance FormPage CommentIdea where
-    type FormPagePayload CommentIdea = Document
+    type FormPagePayload CommentIdea = CommentContent
     type FormPageResult CommentIdea = Comment
 
     formAction (CommentIdea idea mcomment) = U.commentOrReplyIdea idea mcomment
 
     redirectOf (CommentIdea idea _) = U.viewIdeaAtComment idea . view _Id
 
     makeForm CommentIdea{} =
-        "comment-text" .: (Markdown <$> DF.text Nothing)
+        -- TODO: Translation
+        "comment-text" .: (CommentContent <$> validateMarkdown "Comment" (DF.string Nothing))
```

Comment:
(The correct translation of that word is "suggestion for improvement", and it's part of the didactic concept to use that word instead of "Kommentar".  "you all smell bad" is also a valid "Kommentar", but doesn't suggest an improvement. :-)